app/retail/views.py

```diff
@@ -218,6 +217,14 @@ def contributor_landing(request, tech_stack):
         'tech_stack': tech_stack,
     }
 
+    #get activity feed
+    num_activities = 5
```

Comment:
We could probably use a `Paginator` here.  Not sure if we're concerned with that right now.

Reply:
yeah, i dont think its an issue for v1. if we ever want to show more than 10 activity feed itemes it might be an issue